pkgs/development/compilers/elm/packages/elm-review/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildNpmPackage
+, fetchFromGitHub
+}:
+
+buildNpmPackage rec {
+  pname = "elm-review";
+  version = "2.12.0";
+
+  src = fetchFromGitHub {
+    owner = "jfmengels";
+    repo = "node-elm-review";
+    rev = "v${version}";
+    hash = "sha256-+RF1D1gTsZlxE+Pe29T11TT6bWkt17bX7MX0YjBUPcI=";
+  };
+
+  npmDepsHash = "sha256-suNOrGhHoAopBnIar8+HNUk4xwDtFq1gj3GvN2tD5l0=";
+
+  postPatch = ''
+    sed -i '/elm-tooling install/d' package.json
+  '';
+
+  dontNpmBuild = true;
+
+  meta = {
+    changelog = "https://github.com/jfmengels/node-elm-review/blob/v${src.rev}/CHANGELOG.md";
```

Comment:
```suggestion
    changelog = "https://github.com/jfmengels/node-elm-review/blob/${src.rev}/CHANGELOG.md";
```

Reply:
This packages seems to use the `v` prefix: https://github.com/jfmengels/node-elm-review/releases

https://github.com/jfmengels/node-elm-review/blob/2.12.0/CHANGELOG.md returns 404.

Reply:
that is correct. ignore this recommendation

Reply:
`src.rev` already contains the leading v